CMakeLists.txt

```diff
@@ -89,7 +89,8 @@ find_package (Unwind)
 
 if (Unwind_FOUND)
   set (HAVE_LIB_UNWIND 1)
-  set (HAVE_UNWIND_H 1)
+else (Unwind_FOUND)
+  check_include_file_cxx (unwind.h HAVE_UNWIND_H)
```

Comment:
I still do not understand the change. `FindUnwind.cmake` can already tell if `unwind.h` is present.